data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg

```diff
@@ -343,6 +344,14 @@
             id="Ro'Sothian"
         [/recall]
 
+        [recall]
+            id=Varem
```

Comment:
Who is Varem? Player NR like 7 times over the past 10 years and I do not know this character.

Reply:
one of the loyal dwarves who joins you when exploring Malifor's dungeons; really it could be any of them, but I just went with him because he's who I had available

Reply:
I checked the code and he indeed does appear but the acquisition process is such that most players would not have him unless they prefer battling spiders.

Reply:
> If there's multiple loyal dwarves, you could just have it recall one of them at random? The recall tag can take an arbitrary filter.

That's a good idea, actually

Reply:
OK, I changed it to recall an arbitrary loyal dwarf in [0149764](https://github.com/wesnoth/wesnoth/pull/6057/commits/014976436016a4f09260a5e12fecc066999ed833) (untested)